src/Tools/Test Framework/Test Libraries/Any/src/Any.Codeunit.al

```diff
@@ -238,14 +238,28 @@ codeunit 130500 "Any"
         Randomize(Seed);
     end;
 
+    /// <summary>
+    /// Provides the Seed currently used for Pseudo-random number generation.
+    /// </summary>
+    /// <returns>Current seed used for pseudo-random numbers</returns>
+    procedure GetSeed(): Integer
+    begin
+        exit(Seed);
+    end;
+
     /// <summary>.
     /// Sets the default Seed for Pseudo-random number generation (no. of milliseconds since midnight of today).
     /// Setting this value will change the numbers returned.
     /// </summary>
     procedure SetDefaultSeed()
+    var
+        DateTime1, DateTime2 : DateTime;
     begin
+        DateTime1 := CreateDateTime(Today(), 000000T);
+        DateTime2 := CurrentDateTime();
+
         SeedSet := true;
-        Randomize();
+        SetSeed(DateTime2 - DateTime1);
```

Comment:
The default seed is the the current time (# ms since midnight). Date doesn't come into play afaik so why not simply do:
        SetSeed(Time() - 0T);

Reply:
What do you think, @TKapitan. Do you want to make that change?

Reply:
> The default seed is the the current time (# ms since midnight). Date doesn't come into play afaik so why not simply do: SetSeed(Time() - 0T);

You got me. I somehow forgot that anything that returns Duration can be used. Thanks @grobyns